src/services/ErrorMessageService.ts

```diff
@@ -0,0 +1,40 @@
+
+import * as Discord from 'discord.js';
+
+import { AutoWired, Singleton } from 'typescript-ioc';
+import { BaseService } from '../base/BaseService';
+import { DMChannel, GroupDMChannel, TextChannel } from 'discord.js';
+
+@Singleton
+@AutoWired
+export class ErrorMessageService extends BaseService {
+
+  public async init(client) {
+    await super.init(client);
+  }
+
+  /**
+   * Send an error message in a given channel and then delete the message after
+   * a little while.
+   *
+   * @param channel The channel to send the message in.
+   * @param errorText The text of the error message.
+   */
+  public async sendErrorMessage(channel: TextChannel | DMChannel | GroupDMChannel, errorText: string) {

```

Comment:
Just reiterating sei's note for adding the parameter to the send error message function